PowerballApi/Global.asax.cs

```diff
@@ -1,12 +1,22 @@
-﻿namespace PowerballApi.Api
+﻿using System.Reflection;
+using Autofac;
+using Autofac.Integration.WebApi;
+using PowerballApi.Api.Helpers.Cacher;
```

Comment:
Can we move these within the namespace, too?

Reply:
Looks like i don't even need them! Removing from the file.